ouroboros-consensus-test/ouroboros-consensus-test.cabal

```diff
@@ -228,6 +228,7 @@ test-suite test-storage
                        Test.Ouroboros.Storage.ChainDB.Model.Test
                        Test.Ouroboros.Storage.ChainDB.Paths
                        Test.Ouroboros.Storage.ChainDB.StateMachine
+                       Test.Ouroboros.Storage.ChainDB.StateMachine.RunOnRepl
```

Comment:
nitpick: Why not just Test.Ouroboros.Storage.ChainDB.StateMachine.REPL

Reply:
My thought: it's not a REPL, but rather imports to allow the state machine to be run on the REPL. What do you think?